### PR TITLE
Update turn-off-microsoft-usage-data-collection-windows.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details about what data is being collected.

Fixes#https://github.com/MicrosoftDocs/memdocs/issues/3646

### CODE REVIEW

File: memdocs/intune/user-help/turn-off-microsoft-usage-data-collection-windows.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ This article describes how to prevent Microsoft from collecting data about your
 
 When set to **Yes**, your in-app performance and usage data is automatically collected and shared with Microsoft. This type of data helps us improve the reliability and performance of Microsoft products. If you disable the usage data collection, your organization cannot change your preferences.    
 
+For more information on what data is being collected, check [Optional diagnostic data from Intune Client apps](/mem/intune/protect/client-apps-optional-data)
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/turn-off-microsoft-usage-data-collection-windows.md:50]:
```suggestion
For more information about the data that we collect, see [Optional diagnostic data from Intune Client apps](/mem/intune/protect/client-apps-optional-data).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by lenewsad** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**